controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -1168,6 +1171,13 @@ int cli_bml::bml_channel_selection_caller(int numOfArgs)
     }
     return -1;
 }
+int cli_bml::bml_message_to_radio_caller(int numOfArgs)
+{
+    if (numOfArgs == 2) {
```

Comment:
this will never work , you should expect only 1 parameter

Reply:
Big woopsie..